cmd/vic-machine/create/create.go

```diff
@@ -1389,6 +1417,8 @@ func (c *Create) Run(clic *cli.Context) (err error) {
 
 	log.Infof("Initialization of appliance successful")
 
+	// We must check for the volume stores that are present after the portlayer presents.
```

Comment:
wondering if it will be better to move the comments into CheckDockerAPI? I think you extended that method to check docker volume